app/forms/steps/postcode_screen/children_postcodes_form.rb

```diff
@@ -0,0 +1,19 @@
+module Steps
+  module PostcodeScreen
+    class ChildrenPostcodesForm < BaseForm
+      include SingleQuestionForm
```

Comment:
You can remove this include, as the `SingleQuestionForm` is only intended for steps with a Yes/No question, but as this step is a text area, it doesn't apply really.
And once you remove it, instead of using `record_to_persist`, you have to use `c100_application`
Take this other form object as an example as it should be almost identical: https://github.com/ministryofjustice/c100-application/blob/master/app/forms/steps/application/details_form.rb

Reply:
done